parser/ast/expressions.go

```diff
@@ -905,7 +905,7 @@ func (n *PatternLikeExpr) Restore(ctx *format.RestoreCtx) error {
 		return errors.Annotate(err, "An error occurred while restore PatternLikeExpr.Pattern")
 	}
 
-	escape := string(n.Escape)
+	escape := string(n.Escape.(ValueExpr).GetString()[0])
```

Comment:
`escape` is ExprNode now, we cannot assume it is a `valueExpr`. see how `LimitClause` restore.